sampleApp/sample.tsx

```diff
@@ -19,6 +20,15 @@ const render = (globalProps: GlobalProps, notebooks: Notebook[]) => {
 	);
 };
 
+const renderDropdown = (globalProps: GlobalProps, notebooks: Notebook[], label: string, popupVisible: boolean) => {
+	ReactDOM.render(
+		<OneNotePickerDropdown globals={globalProps.globals} notebooks={notebooks} dropdownLabel={label} popupDirection={'bottom'} popupVisible={popupVisible}/>,
+		document.getElementById('oneNotePickerDropdown') as HTMLElement
+	);
+};
+
+export let defaultDropdownLabel = "";
```

Comment:
See above, this would mean we don't need this variable since we always derive the label from props.